kevm-pyk/src/kevm_pyk/foundry.py

```diff
@@ -343,7 +345,9 @@ def foundry_prove(
             target_term = KDefinition__expand_macros(foundry.kevm.definition, target_term)
             kcfg.replace_node(kcfg.get_unique_target().id, CTerm(target_term))
             if simplify_init:
-                with KCFGExplore(foundry.kevm, port=find_free_port(), bug_report=br) as kcfg_explore:
+                with KCFGExplore(
+                    foundry.kevm, port=find_free_port(), bug_report=br, kore_rpc_command=kore_rpc_command
```

Comment:
Added SMT options to `__init__`: https://github.com/runtimeverification/pyk/pull/276

Reply:
Also, `find_free_port` can be dropped now: https://github.com/runtimeverification/pyk/pull/262

Reply:
Done.